src/ui_parts/tag_editor.gd

```diff
@@ -39,8 +39,7 @@ func _ready() -> void:
 	RenderingServer.canvas_item_set_z_index(surface, 1)
 	title_label.text = tag.name
 	Utils.set_max_text_width(title_label, 180.0, 0.0)  # Handle TagUnknown gracefully.
-	title_icon.texture = unknown_icon if tag is TagUnknown\
-			else load("res://visual/icons/tag/" + tag.name + ".svg")
+	title_icon.texture = tag.icon
```

Comment:
Doesn't this mean unknown_icon no longer needs to be a variable in tag_editor.gd?